src/lcf/rooms.395

```diff
@@ -55,7 +54,7 @@
 		      <TELL ,FLUSHSTR1>
 		      <QUIT>)>
 	       <COND (<AND <TYPE? <SET STV <STARTER>> FIX>
-			   <G=? .STV 3>>
+			   <G=? .STV 6>>
```

Comment:
What is this?

Reply:
This basically neuters the calculation in STARTER. This function is supposed to return a small number if the user is allowed to run Zork and a larger number if not. There are various checks. But the last check appears to be trying to perform a calculation on pure memory pages (I think) that was always preventing the running of Zork (returning 5). I couldn’t figure out why, so I raised the limit from 3 to “greater than 5”.

My thinking was that I’d try to understand and tackle that problem later.